definitions/variable/industry/tag_demand_sectors.yaml

```
@@ -0,0 +1,27 @@
+- Demand Sector:
+    - Electricity:
+        description: electricity generation sector
+    - Electricity|Biomass:
+          description: electricity generation with biomass power plants
+    - Electricity|Coal:
+          description: electricity generation with coal power plants
+    - Electricity|Gas:
+          description: electricity generation with gas power plants
+    - Electricity|Hydro:
+          description: electricity generation with hydro power plants
+    - Electricity|Nuclear:
+          description: electricity generation with nuclear power plants
+    - Electricity|Solar:
+          description: electricity generation with solar PV
+    - Electricity|Wind:
+          description: electricity generation with wind turbines
+    - Electricity|Transmission and Distribution:
+          description: electricity transmission and distribution
+    - Transportation:
+        description: transportation sector excluding infrastructure (e.g. fixed structures, installations and networks)
+    - Residential:
+        description: residential sector
+    - Commercial:
+        description: commercial sector
+    - Infrastructure|Mobility:
```

I'm hesitant to introduce a new term "Mobility", which might cause more confusion than it helps to avoid ambiguity. So I suggest to make that an explicit sub-category of "Transportation" instead.
```suggestion
    - Transportation|Infrastructure:
```

---

I don't have a strong feeling about 'Mobility' vs. 'Transportation', but would suggest to keep the order 'Infrastructure|Transportation' as there are other infrastructure types (e.g., water) that could be added.

---

I also considered that, but we also have "Electricity|Transmission and Distribution" (in the same file), so there, infrastructure is a sub-component of electricity. Not sure which other possible types of infrastructure we have beyond energy, transportation, and water? Buidlings? Telecom?